input/fsh/models.fsh

```diff
@@ -21,8 +21,10 @@ Expression: "(path | from | forEach | forEachOrNull | union).count() = 1"
 Logical: ViewDefinition
 Title: "View Definition"
 Description: """
-View definitions are a tabular projection of a FHIR resource, where the columns and inclusion 
+View definitions represent a tabular projection of a FHIR resource, where the columns and inclusion 
 criteria are defined by FHIRPath expressions. 
+
+See the [View Definition page](view-definition.html) for details.
```

Comment:
I don’t know about adding this URL as relative… Does IG Publisher make this an absolute URL?

Reply:
Thanks, I went ahead and made this an absolute URL as suggested.